assets/src/edit-story/elements/gif/output.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import StoryPropTypes from '../../types';
+import MediaOutput from '../media/output';
+
+function defaultForUndefined(value, def) {
+  return value === undefined ? def : value;
+}
+
+function GifOutput({ element, box }) {
+  const { resource } = element;
+
+  const sourceProps = {
+    type: resource.output.mimeType,
+    src: resource.output.src,
+  };
+
+  const props = {
+    autoPlay: 'autoplay',
+    poster: '',
```

Comment:
Don't we need a poster? Or is that left as a future enhancement?

Reply:
I'll always prefer to do things incrementally. :) But also per [the issue's discussion](https://app.zenhub.com/workspaces/web-stories-5e94d3aced449034e1e9b226/issues/google/web-stories-wp/4782#issuecomment-708660370) this is functional for the release, and we don't yet get a poster image from the Tenor/media3p google api.